internal/errutil/error_test.go

```diff
@@ -0,0 +1,22 @@
+package errutil
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCephError(t *testing.T) {
+	radosErr := cephErrno(107)
+	assert.Equal(t, "Transport endpoint is not connected", radosErr.Error())
+
+	cephFSErr := GetError("cephfs", 2)
+	assert.Equal(t, "cephfs: ret=2, No such file or directory",
+		cephFSErr.Error())
+	assert.Equal(t, 2, cephFSErr.(cephError).ErrorCode())
+
+	rbdErr := fmt.Errorf("RBD image not found: %w", GetError("rbd", 2))
+	assert.True(t, errors.Is(cephFSErr, errors.Unwrap(rbdErr)))
```

Comment:
do you need the `errors.Unwrap()` here?

Reply:
I think it works if we reverse the order as follows:
`assert.True(t, errors.Is(rbdErr, cephFSErr))`

But may be we take this opportunity to test `Unwrap()` also?

Reply:
It is nice to test `Unwrap()`, sure. But the main test should also show that `errors.Is(cephFSErr, rbdErr)` works, which is closer to what applications would do with `errors.Is(cephFSErr, rados.ErrNotFound)`.

Reply:
Maybe test `Unwrap()` like this:
```go
assert.True(t, errors.Unwrap(cephFSErr) == errors.Unwrap(rbdErr)))
```

Reply:
Cool, updated.